src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -604,6 +605,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CustomLinkerArg Include="-s MAXIMUM_MEMORY=$(EmccMaximumHeapSize)" Condition="'$(EmccMaximumHeapSize)' != ''" />
       <CustomLinkerArg Include="-s INITIAL_MEMORY=$(EmccInitialHeapSize)" Condition="'$(EmccInitialHeapSize)' != ''" />
       <CustomLinkerArg Include="-s ASSERTIONS=1" Condition="'$(EmccEnableAssertions)' == 'true'" />
+      <CustomLinkerArg Condition="'$(WasmEnableThreads)' == 'true'" Include="-pthread" />
```

Comment:
I saw earlier Emscripten gives some warning like "JS will be slow with memory growth + threads", seems we should disable it.

Reply:
disabled growth and `MAXIMUM MEMORY` for threads.

Reply:
Hmm, I only meant to disable the warning. I don't think we can disable memory growth...

Reply:
lol, I was thinking we would block users from running slow code.